src/palindrom-dom.js

```diff
@@ -236,24 +211,9 @@ const PalindromDOM = (() => {
     }
   }
 
-  PalindromDOM.prototype = Object.create(Palindrom.prototype);
```

Comment:
Aren't you killing inheritance from Palindrom with this?

Reply:
Not really. It was redundant. `extends Palindrom` replaces it.